tests/testthat/test-conditional-transform.R

```diff
@@ -13,9 +13,28 @@ with_mock_crunch({
                 " .*formulas.* argument, or through .*....*"
             )
         )
+               
+        expect_error(
```

Comment:
Do we maybe need an integration test to ensure that the new type of transform goes through okay?